sari/.htaccess

```diff
@@ -1,23 +1,20 @@
-## Turn off MultiViews
+Options +FollowSymLinks
+RewriteEngine on
+
+# Turn off MultiViews
 Options -MultiViews
 
-# Rewrite engine setup
-RewriteEngine On
-			
-# Rewrite rule to serve HTML content
-RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
-RewriteCond %{HTTP_ACCEPT} text/html [OR]
-RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
-RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^$ https://swissartresearch.net/ontology/sari [R=303,L]
+AddType text/turtle .ttl
+
+# In case of accept header <text/turtle>
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+RewriteRule ^$ https://schema.swissartresearch.net/ontology/sari.ttl [R=303,NE,L]
 
-# Rewrite rule to serve RDF/XML content if requested
-RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
-RewriteCond %{HTTP_ACCEPT} application/owl\+xml
-RewriteRule ^$ https://swissartresearch.net/ontology/sari.rdfs.xml [R=303,L]
+# If suffix ttl, redirect to turtle version
+RewriteRule ^sari.ttl$ https://schema.swissartresearch.net/ontology/sari.ttl [R=302,NE,L]
```

Comment:
Here are you expecting https://w3id.org/sari/sari.ttl?
Or https://w3id.org/sari.ttl ? Because you are specifying the former